src/plugins/auto/src/plugin_config.hpp

```diff
@@ -17,7 +17,7 @@
 namespace MultiDevicePlugin {
 using namespace InferenceEngine;
 // legacy config
-static constexpr ov::Property<bool, ov::PropertyMutability::RW> exclusive_asyc_requests{"EXCLUSIVE_ASYNC_REQUESTS"};
+static constexpr ov::Property<bool, ov::PropertyMutability::RW> exclusive_async_requests{"EXCLUSIVE_ASYNC_REQUESTS"};
```

Comment:
it's going to be removed in [another ](https://github.com/openvinotoolkit/openvino/pull/16840) PR. Don't change AUTO